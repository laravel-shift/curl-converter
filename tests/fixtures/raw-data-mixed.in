curl --request POST 'https://api.com' --header 'Accept: application/json' --data-raw 'some=data' -d foo=bar
