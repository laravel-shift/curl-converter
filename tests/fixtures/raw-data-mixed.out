Http::asForm()
    ->acceptJson()
    ->post('https://api.com', [
        'some' => 'data',
        'foo' => 'bar',
    ]);
